docker/Makefile

```
@@ -0,0 +1,36 @@
+# Makefile - Convenient commands
+
+.PHONY: dev prod build clean logs health
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)